src/vcard/mod_vcard.erl

```diff
@@ -246,7 +236,22 @@ config_spec() ->
                  <<"ldap_binary_search_fields">> => #list{items = #option{type = binary,
                                                                           validate = non_empty}},
                  <<"riak">> => riak_config_spec()
-                }
+                },
+       defaults = #{<<"iqdisc">> => parallel,
+                    <<"host">> => mongoose_subdomain_utils:make_subdomain_pattern("vjud.@HOST@"),
+                    <<"search">> => true,
+                    <<"backend">> => mnesia,
+                    <<"matches">> => 30,
+                    <<"ldap_pool_tag">> => default,
+                    <<"ldap_uids">> => [{<<"uid">>, <<"%u">>}],
+                    <<"ldap_vcard_map">> => mod_vcard_ldap:default_vcard_map(),
+                    <<"ldap_search_fields">> => mod_vcard_ldap:default_search_fields(),
+                    <<"ldap_search_reported">> => mod_vcard_ldap:default_search_reported(),
+                    <<"ldap_search_operator">> => 'and',
+                    <<"ldap_binary_search_fields">> => []
```

Comment:
Just a note: how about putting all these options in a nested map `ldap` and treating them like `riak`? I know it's a change in config structure, but for me it looks like a change for the better? It could be done in a separate PR.

Reply:
I was thinking about it, and even some LDAP options are shared between different modules using LDAP. There could be a section defined in the `mongoose_ldap_config` with the common options.

Reply:
With the change, the options that the user would specify would be, for example `modules.mod_vcard.ldap.pool_tag` instead of `modules.mod_vcard.ldap_pool_tag`, correct?

Reply:
Yes, I think we could do it separately, in MAM I could also move the options into subsections, but it would be too much of a change in one go.

Reply:
Yeah, I'd rather do it separately as well